### ISSUE TITLE
Does database get deleted when the user delete the app ?

### ISSUE THREAD

**Issue by MisterJack49** [brody2consult/cordova-sqlite-storage#258]
Hi,

Does a database get deleted when the user delete the app ?

**Comment by nolanlawson** [comment]
Yes, unless it's iOS and you're using the `location` option to back it up to iCloud. Even then, it's deleted from the device; it just still exists in iCloud.
